x/src/test.rs

```diff
@@ -100,6 +151,60 @@ pub fn run(mut args: Args, config: Config) -> Result<()> {
             all_features,
             ..base_args
         })?;
-        Ok(())
     }
+
+    let mut debug_dir = project_root();
+    debug_dir.push("target/debug/");
+
+    if args.lcovdir.is_some() {
+        let mut grcov_lcov = Command::new("grcov");
+        grcov_lcov
+            .arg(debug_dir.as_os_str())
+            .arg("-t")
+            .arg("lcov")
+            .arg("--llvm")
+            .arg("--branch")
+            .arg("--ignore")
+            .arg("/*")
+            .arg("--ignore")
+            .arg("x/*")
+            .arg("--ignore")
+            .arg("testsuite/*")
+            .arg("-o")
+            .arg(args.lcovdir.unwrap());
+        info!("{:?}", grcov_lcov);
+        grcov_lcov.stdout(Stdio::inherit()).stderr(Stdio::inherit());
+        let output = grcov_lcov.output()?;
+        output.status.success();
+    };
```

Comment:
Needs some testing.   Speaking of -- need to start looking at unit tests to see what I can add, as this gets more complicated.